plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -1,27 +1,54 @@
 /* global wp, woocommerce_admin_meta_boxes_variations, woocommerce_admin, accounting */
-jQuery( function( $ ) {
-    'use strict';
+jQuery( function ( $ ) {
+	'use strict';
 
 	/**
 	 * Variations actions
 	 */
 	var wc_meta_boxes_product_variations_actions = {
-
 		/**
 		 * Initialize variations actions
 		 */
-		init: function() {
+		init: function () {
 			$( '#variable_product_options' )
-				.on( 'change', 'input.variable_is_downloadable', this.variable_is_downloadable )
-				.on( 'change', 'input.variable_is_virtual', this.variable_is_virtual )
-				.on( 'change', 'input.variable_manage_stock', this.variable_manage_stock )
+				.on(
+					'change',
+					'input.variable_is_downloadable',
+					this.variable_is_downloadable
+				)
+				.on(
+					'change',
+					'input.variable_is_virtual',
+					this.variable_is_virtual
+				)
+				.on(
+					'change',
+					'input.variable_manage_stock',
+					this.variable_manage_stock
+				)
 				.on( 'click', 'button.notice-dismiss', this.notice_dismiss )
 				.on( 'click', 'h3 .sort', this.set_menu_order )
+				.on(
+					'click',
+					'button.add_price_for_variations',
+					this.open_modal_to_set_variations_price
```

Comment:
Just underlining the lines added to this file. The lint modified a lot of things here.